src/apps/tezos/writers.py

```diff
@@ -0,0 +1,24 @@
+from apps.common.writers import (
```

Comment:
as noted in the baking PR, this file should not exist.

please use the `write_uint16_be` implementation from your baking PR, and move `write_bool` to helpers
in fact, this is almost certainly going in sooner than baking support, so you can then use this as the basis of the baking PR